common/src/main/java/com/wynntils/services/mapdata/attributes/MapAttributesBuilder.java

```diff
@@ -106,17 +108,17 @@ public MapAreaAttributesBuilder asAreaAttributes() {
 
     protected void checkInvalidAttribute(String fieldName) {
         // FIXME: WaypointLocation is giving Unsupported attribute set: borderWidth when using WaypointCreationScreen
```

Comment:
If you want to try out this pr, you will need to comment out this method. Fix can either be pushed to this branch or the main MapData one and will just update this branch to have the fix

Reply:
It's fixed in my latest mapdata pr.